Cesium3DTilesContent/src/upsampleGltfForRasterOverlays.cpp

```diff
@@ -1134,6 +1158,9 @@ static void addSkirts(
       edgeIndices.north.end(),
       sortEdgeIndices.begin(),
       [](const EdgeVertex& v) { return v.index; });
+  if (hasInvertedVCoordinate) {
+    std::reverse(sortEdgeIndices.begin(), sortEdgeIndices.end());
```

Comment:
Is it possible to eliminate the `std::reverse` operation by just swapping the iterators in `std::transform` or inverting the comparison in the `std::sort`?